terminal/tcell/cell_options.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Google Inc.
```

Comment:
What are your thoughts on updating the wiki as discussed in #223? Will you have the time and will to do that or would you prefer if I do?

Reply:
Sure I can probably update the wiki, at least after this PR is merged

Reply:
Doing this after we merge it in sounds good to me, thank you for offering and I am happy to help you with that, just let me know.